app/src/main/java/com/google/samples/apps/sunflower/compose/home/HomeScreen.kt

```diff
@@ -61,9 +73,55 @@ enum class SunflowerPage(
     PLANT_LIST(R.string.plant_list_title, R.drawable.ic_plant_list_active)
 }
 
+@Composable
+fun HomeScreen(
+    onPlantClick: (Plant) -> Unit,
+    viewModel: PlantListViewModel = hiltViewModel()
+) {
+    val activity = (LocalContext.current as AppCompatActivity)
+
+    val menuProvider = object : MenuProvider {
+        override fun onCreateMenu(menu: Menu, menuInflater: MenuInflater) {
+            menuInflater.inflate(R.menu.menu_plant_list, menu)
+        }
+
+        override fun onMenuItemSelected(menuItem: MenuItem): Boolean {
+            return when (menuItem.itemId) {
+                R.id.filter_zone -> {
+                    if (viewModel.isFiltered()) {
+                        viewModel.clearGrowZoneNumber()
+                    } else {
+                        viewModel.setGrowZoneNumber(9)
+                    }
+                    true
+                }
+                else -> false
+            }
+        }
+    }
+
+    AndroidViewBinding(HomeScreenBinding::inflate)
```

Comment:
Not sure I understand why you need the AndroidViewBinding here - can't you just add HomePagerScreen composable directly. Is it cause of `setSupportActionBar`?

Reply:
Yep - it's because of `setSupportActionBar`. Though generally, it's because View interop is required to support collapsing toolbars in M2.